### PR TITLE
change description format within tables

### PR DESCRIPTION
**Pull request by pnaik1**
https://issues.redhat.com/browse/RHOAIENG-11499

## Description
Removed the separate description column and  have the description underneath the resource name for the experiments table. and archive experiment table

Screenshot 2024-09-11 at 10 54 50 AM

Screenshot 2024-09-11 at 10 54 30 AM

## How Has This Been Tested?

1. Go to the experiment Page 
2. Check whether the description column is not present
3. check whether the description is present underneath the resource name and gets truncated when > 2 lines

## Test Impact
Added cypress test

## Request review criteria:

Self checklist (all need to be checked):
- [x] The developer has manually tested the changes and verified that the changes work
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

**Comment by pnaik1** [comment]
Added a `width` to the Experiment column row, since after having the description underneath the resource was causing unnecessary extra space to the right

File: frontend/src/concepts/pipelines/content/tables/experiment/ExperimentTableRow.tsx [hunk 1]
```diff
@@ -28,20 +28,26 @@ const ExperimentTableRow: React.FC<ExperimentTableRowProps> = ({
   return (
     <Tr>
       <CheckboxTd id={experiment.experiment_id} isChecked={isChecked} onToggle={onToggleCheck} />
-      <Td dataLabel="Experiment">
-        <Link
-          to={
-            isArchived
-              ? experimentArchivedRunsRoute(namespace, experiment.experiment_id)
-              : experimentRunsRoute(namespace, experiment.experiment_id)
+      <Td dataLabel="Experiment" width={40}>
```
Comment (DaoDaoNoCode) [inline frontend/src/concepts/pipelines/content/tables/experiment/ExperimentTableRow.tsx:31]: You can give the `width` in "frontend/src/concepts/pipelines/content/tables/columns.ts" for every column like other tables to get a better layout. Just make sure the sum of all the column widths is 100.

**Review by DaoDaoNoCode — APPROVED** [review]
/lgtm
Screenshot 2024-09-12 at 10 15 38 AM

**Pull request merged**